src/UserSettingsStore.js

```diff
@@ -149,6 +149,23 @@ module.exports = {
         return MatrixClientPeg.get().setAccountData("im.vector.web.settings", settings);
     },
 
+    getLocalSettings: function() {
```

Comment:
it would be good to explain wtf a 'local setting' is. Turns out, it's a setting which is persisted in localstorage.